src/main/java/fr/insee/genesis/controller/rest/ResponseController.java

```diff
@@ -70,7 +70,7 @@ public ResponseEntity<Object> saveResponsesFromXmlFile(     @RequestParam("pathL
         }
         List<SurveyUnitUpdateDto> suDtos = new ArrayList<>();
         for (LunaticXmlSurveyUnit su : campaign.getSurveyUnits()) {
-            suDtos.addAll(LunaticXmlAdapter.convert(su, variablesMap, campaign.getIdCampaign()));
+            suDtos.addAll(LunaticXmlAdapter.convert(su, variablesMap, campaign.getIdCampaign(), Mode.WEB));
```

Comment:
Why WEB ?

Reply:
It was code for testing purpose that stayed in. Now the mode is required in the endpoint